tests/tests_geomstats/test_wald_space.py

```diff
@@ -0,0 +1,246 @@
+import geomstats.tests
+from geomstats.stratified_geometry.wald_space import Split, Topology, Wald, WaldSpace
+from tests.conftest import Parametrizer
+from tests.data.wald_space_data import WaldSpaceTestData, WaldTestData
+from tests.stratified_geometry_test_cases import PointSetTestCase, PointTestCase
+
+
+class TestWaldSpace(PointSetTestCase, metaclass=Parametrizer):
+
+    _PointSet = WaldSpace
+    testing_data = WaldSpaceTestData()
+
+
+class TestWald(PointTestCase, metaclass=Parametrizer):
```

Comment:
Do not forget to add tests to (at least) all public methods: `e.g. `Wald.generate_wald`. Sometimes the tests are trivial, but at least they force as to keep the API over time.